### PR TITLE
Add a helper component for table layouts

### PR DESCRIPTION
**Pull request by mperrotti**
This came out of [DCOS-60082](https://jira.mesosphere.com/browse/DCOS-60082). I have a branch where I've resolved that ticket using the components in this PR.

### CODE REVIEW

**Comment by GeorgiSTodorov** [comment]
I am getting a weird horizontal scroll in some cases. Tested in both Firefox and Chrome.
Peek 2019-10-18 09-27

**Comment by mperrotti** [comment]
@GeorgiSTodorov - I can't reproduce this in Kommander when using the TableView component, just in Storybook, and only when I resize my window slowly. Do you think this is a blocker?

**Comment by GeorgiSTodorov** [comment]
It's not a blocker, it just caught my attention.

File: packages/tableView/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- have a minimum height so they don't disappear on very short viewports
+
+## Components
+### TableView
+Creates a flex [formatting context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flow_Layout/Intro_to_formatting_contexts). This ensures the `TableViewBody` component fills the available height.
+
+The height of a `TableView`'s parent container will dictate the height of the `TableView`.
+
+### TableViewHeader
+Contains the content that appears above the table. It's height depends on it's children.
```
Comment (GeorgiSTodorov) [inline packages/tableView/README.md]: Shouldn't this say "its"?

File: packages/tableView/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+### TableViewBody
+Contains the table, and grows to fill up the available height inside of `TableView` after factoring in the height of the `TableViewHeader` and `TableViewFooter`.
+This component can be used to specify a minimum height for the table, measured in number of rows.
+For best results, only pass 1 child to this element.
+
+### TableViewFooter
+Contains the content that appears below the table. It's height depends on it's children.
```
Comment (GeorgiSTodorov) [inline packages/tableView/README.md]: Same here, I think it should be "its".

**Pull request merged**